source/extensions/filters/network/redis_proxy/command_splitter_impl.cc

```diff
@@ -31,15 +31,25 @@ void SplitRequestBase::onWrongNumberOfArguments(SplitCallbacks& callbacks,
       fmt::format("wrong number of arguments for '{}' command", request.asArray()[0].asString())));
 }
 
+void SplitRequestBase::updateStats(const bool success) {
```

Comment:
I factored this in to a separate method because, I want to follow-up with latency stats as well. This will help to add latency metric in the future PR